kombu/transport/redis_cluster.py

```diff
@@ -173,8 +173,8 @@ def _register(self, channel, client, conn, cmd):
                         node = channel.consumer_client.get_node_from_key(conn.key)
                     if node:
                         break
-                except Exception as e:
-                    logger.error('Error while getting node from key', extra={"e": e, "key": conn.key})
+                except:
+                    logger.exception('Error while getting node from key', extra={"key": conn.key})
```

Comment:
Just for curious, may I ask why the exception data has been excluded from the logging?

Reply:
https://docs.python.org/3/library/logging.html#logging.Logger.exception `logger.exception` automatically adds exception information

Reply:
Oh I see. Thanks for the explanation